tests/e2e/cucumber/facets.spec.ts

```diff
@@ -0,0 +1,100 @@
+import { Then, When } from 'cypress-cucumber-preprocessor/steps';
+import '../cucumber/global-definitions';
+
+/**
+ * Click on a filter from a certain facet.
+ *
+ * @param facetName - Name of the facet which the filter to be clicked belongs to.
+ * @param nthFilter - Position of the filter to be clicked.
+ *
+ * @internal
+ */
+function clickFacetNthFilter(facetName: string, nthFilter: number): void {
+  cy.getByDataTest(facetName)
+    .getByDataTest('base-filters-item')
+    .eq(nthFilter)
+    .click('top')
```

Comment:
I was having some troubles when it came to select the parent filter of the hierarchical one since I was using `base-filters-item`. Now it should be fine.